contracts/ronin/gateway/RoninBridgeManager.sol

```diff
@@ -35,6 +48,32 @@ contract RoninBridgeManager is BridgeManager, BOsGovernanceProposal {
     }
   }
 
+  /**
+   * @dev See `CoreGovernance-_proposeGlobal`.
+   *
+   * Requirements:
+   * - The method caller is governor.
+   *
+   */
+  function proposeGlobal(
+    uint256 _expiryTimestamp,
+    GlobalProposal.TargetOption[] calldata _targetOptions,
+    uint256[] calldata _values,
+    bytes[] calldata _calldatas,
+    uint256[] calldata _gasAmounts
+  ) external onlyGovernor {
+    _proposeGlobal(
+      _expiryTimestamp,
+      _targetOptions,
+      _values,
+      _calldatas,
+      _gasAmounts,
+      address(this),
+      getContract(ContractType.BRIDGE),
```

Comment:
Should be `ContractType.BRIDGE_MANAGER`

Reply:
Previous version we fed in address of RoninGatewayV2.